docs/en/05_domain-configuration.md

```diff
@@ -8,19 +14,19 @@ has its own default locale, sublocales, while still acting as a single multi-dom
 
 For example, your website may have the following URLs locally:
 
- * http://localhost/mysite/ (en_US default locale)
- * http://localhost/mysite/es_US/
- * http://localhost/mysite/es_ES/
- * http://localhost/mysite/zh_cmn/
- * http://localhost/mysite/zh_yue/
+- `http://localhost/mysite/` (en_US default locale)
+- `http://localhost/mysite/es_US/`
+- `http://localhost/mysite/es_ES/`
+- `http://localhost/mysite/zh_cmn/`
+- `http://localhost/mysite/zh_yue/`
 
 When deploying this website to the live environment, you may wish to use the following as the public urls for each:
```

Comment:
```suggestion
When deploying this website to the live environment, you may wish to use the following as the public URLs for each:
```
Interesting... the linter should have picked this up, but I guess it's explicitly looking for `URL` and not `URLs`

Reply:
Updated